pycoast/cw_base.py

```diff
@@ -834,18 +834,18 @@ def add_overlay_from_dict(self, overlays, area_def, cache_epoch=None, background
 
             params = overlays['points'].copy()
 
-            points_list = list(params.pop('list'))
+            points_list = list(params.pop('points_list'))
             font_file = params.pop('font')
             font_size = int(params.pop('font_size', DEFAULT_FONTSIZE))
 
             symbol = params.pop('symbol', DEFAULT_SYMBOL)
-            pt_size = int(params.pop('pt_size', DEFAULT_PTSIZE))
+            ptsize = int(params.pop('ptsize', DEFAULT_PTSIZE))
```

Comment:
why this change? `point_size`?

Reply:
It just make the keys from the config file consistence with the arguments in the function interface to avoid confusion. Similar as changing from 'list' to 'points_list' as we discussed.

Reply:
You changed all of them though. The kwarg here, the argument name in `add_points`, and the key in the config. With this change it isn't consistent with the cities.

Reply:
I think the confusion is form that the keys ini config file for the add_cities() are not consistence with the add_cities() function. like 'list' vs 'citylist'; 'pt_size' vs 'ptsize'. i was trying to make they consistence in the add_points() case. If you think it is better to keep the 'pt_size' in the ini files, I can change them back.